tests/passw_hardening/conftest.py

```diff
@@ -0,0 +1,86 @@
+import pytest
```

Comment:
general:
remove the clean before each test(in each fixture).
assumption is that the setup is clear